client/src/components/ViewPostDialog.tsx

```diff
@@ -115,7 +116,7 @@ function MoreOptions(props: {
       >
         {props.isAuth ? (
           <>
-            <MenuItem onClick={closeMenu}>Edit</MenuItem>
+            <MenuItem onClick={() => props.toggleEdit(true)}>Edit</MenuItem>
```

Comment:
I'm assuming `isAuth` specifies if the user created the post they are trying to delete/update? If so, lets update that variable name, seems like it's just checking of the user is logged in. Something like `userHasCreatedPost`